grafana/staging/dashboards/catalyst-multi-node.json

```diff
@@ -29,6 +29,10 @@
   "liveNow": false,
   "panels": [
     {
+      "datasource": {
+        "type": "prometheus",
+        "uid": "PBFA97CFB590B2093"
```

Comment:
wouldn't the uid be different for staging and prod prometheus data sources?

Reply:
I thought so but looking at other staging files it's `PBFA97CFB590B2093` everywhere.

Reply:
https://eu-metrics-monitoring.livepeer.monster/grafana/datasources